fanuc_lrmate200id_gazebo/package.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>fanuc_lrmate200id_gazebo</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial Gazebo support for the Fanuc LR Mate 200iD (and variants).
+    </p>
+    <p>
+      This package contains configuration data and launch files for Gazebo
+      for Fanuc LR Mate 200iD manipulators.
+    </p>
+  </description>
+  <author>G.A. vd. Hoorn (TU Delft Robotics Institute)</author>
+  <author email="www.kentaro.wada@gmail.com">Kentaro Wada</author>
+  <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn (TU Delft Robotics Institute)</maintainer>
+  <license>BSD</license>
+
+  <url type="website">http://wiki.ros.org/fanuc_lrmate200id_support</url>
+  <url type="bugtracker">https://github.com/ros-industrial/fanuc_experimental/issues</url>
+  <url type="repository">https://github.com/ros-industrial/fanuc_experimental</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <test_depend version_gte="1.9.55">roslaunch</test_depend>
+
+  <exec_depend>control_manager</exec_depend>
+  <exec_depend>fanuc_lrmate200id_support</exec_depend>
+  <exec_depend>gazebo_ros</exec_depend>
+  <exec_depend>robot_state_publisher</exec_depend>
+  <exec_depend>xacro</exec_depend>
```

Comment:
Wouldn't you need at least `position_controllers` and `joint_trajectory_controller` (and maybe others)?